README.md

```diff
@@ -1,328 +1,16 @@
-oneanddone
-==========
+# One and Done
 
-One and Done is written with [Django][django].
+This repository holds code and automation that were used for Mozilla's [One and Done](https://oneanddone.mozilla.org/) website.
```

Comment:
Also, it was written to match https://github.com/mozilla/moz-grid-config/pull/161/files